### PR TITLE
bat: update to 0.16.0.

### PR DESCRIPTION
**Pull request by olafmersmann**
Update bat to version 0.16.0.

### CODE REVIEW

**Comment by Johnnynator** [comment]
You have an unrelated frp commit in here, can you drop it and rebase? thx.

**Comment by ericonr** [comment]
Could you please look into installing the shell completions? They are available from upstream.

**Comment by olafmersmann** [comment]
> Could you please look into installing the shell completions (using `vcompletion`, possibly)? They are available from upstream.

Done.

**Comment by olafmersmann** [comment]
> You have an unrelated frp commit in here, can you drop it and rebase? thx.

Sorry about that. Fixed.

File: srcpkgs/bat/template [hunk 1]
```diff
@@ -24,4 +24,9 @@ post_install() {
 	# The manual page is hidden somewhere deep in the build tree
 	manpage=$(find ${wrksrc}/target -name bat.1)
 	vman ${manpage}
+	# Completions are also hidden somewhere deep in the build tree
+	fish_completion=$(find ${wrksrc}/target -name bat.fish)
+	vcompletion ${fish_completion} fish
+	zsh_completion=$(find ${wrksrc}/target -name bat.zsh)
+	vcompletion ${zsh_completion} zsh
```
Comment (ericonr) [inline srcpkgs/bat/template:31]: Hmm, no bash ones? Shame. Thanks!

Comment (olafmersmann) [reply srcpkgs/bat/template:31]: Sadly no. See https://github.com/sharkdp/bat/tree/master/assets/completions

**Pull request merged**